### PR TITLE
modal box component styles and content arrangement

### PR DESCRIPTION
**Pull request by David0817**
- dashboard/transaction lists/ details modal box
- dashboard/transfer / Transaction confirmation modal box

### CODE REVIEW

File: src/app/App.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default class App extends AppTs { }
 <style lang="less">
 @import "../views/resources/css/common.less";
 @import "../views/resources/css/forms.less";
-// @import "../views/resources/css/ivewWindows.less";
+/*  @import "../views/resources/css/ivewWindows.less"; */
```
Comment (evias) [inline src/app/App.vue]: useless change

File: src/components/TransactionDetails/Transfer/Transfer.less [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      grid-area: b;
+    }
+  
+    .transaction-row-outer-container:nth-child(3) {
+      grid-area: c;
+    }
+  
+    .transaction-row-outer-container:nth-child(4) {
+      grid-area: d;
+    }
```
Comment (evias) [inline src/components/TransactionDetails/Transfer/Transfer.less]: this is not acceptable as is. What happens if any more "areas" are added ?

Please do not used "named areas" like that. If you are required to use `display: grid`, then please use _percentages_ to fill the correct space (and it must add up to 100%, **always**.)

File: src/components/TransactionDetails/Transfer/Transfer.less [hunk 3]
```diff
@@ -0,0 +1,55 @@
+@import "../../../views/resources/css/variables.less";
+.transaction-details-outer-container{
+    max-height: 2.8rem;
+    overflow-y: scroll;
+  }
+  .transaction-details-item-inner-container {
+    display              : grid;
+    grid-row-gap         : .1rem;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows   : 1fr 1fr 1fr;
```
Comment (evias) [inline src/components/TransactionDetails/Transfer/Transfer.less]: why introduce `fr` sizes ? Please use consistent sizing format

File: src/components/TransactionDetails/Transfer/Transfer.vue [hunk 4]
```diff
@@ -78,5 +78,5 @@ export default class Transfer extends Vue {
 </script>
 
 <style lang="less" scoped>
-@import "../TransactionDetails.less";
```
Comment (evias) [inline src/components/TransactionDetails/Transfer/Transfer.vue]: what happens to **common styles** that are put inside TransactionDetails.less ? You should not remove this import.

File: src/views/forms/FormExtendNamespaceDurationTransaction/FormExtendNamespaceDurationTransaction.vue [hunk 5]
```diff
@@ -53,17 +53,17 @@
               :immediate="true"
               slim
             >
-              <input v-show="false" v-model="newDuration" @change="validate">
-              <ErrorTooltip :errors="errors">
+              <!-- <input v-show="false" v-model="newDuration" @change="validate"> -->
```
Comment (evias) [inline src/views/forms/FormExtendNamespaceDurationTransaction/FormExtendNamespaceDurationTransaction.vue]: why removing this input? it is breaking the namespace extension form.

Comment (vincent-lee90) [reply src/views/forms/FormExtendNamespaceDurationTransaction/FormExtendNamespaceDurationTransaction.vue]: I commit the code by accident

File: src/components/TransactionDetails/Transfer/Transfer.vue (hunk 4 above)
Comment (vincent-lee90) [reply src/components/TransactionDetails/Transfer/Transfer.vue]: the transfer should has own styles  instead of  the other component styles.

Comment (vincent-lee90) [reply src/components/TransactionDetails/Transfer/Transfer.vue]: and it has already cause some style conflict when i reform the modal style.

File: src/components/TransactionDetails/Transfer/Transfer.less (hunk 3 above)
Comment (vincent-lee90) [reply src/components/TransactionDetails/Transfer/Transfer.less]: sorry,I just want to know there is any problem about fr ?

**Comment by evias** [comment]
Closing this PR ; there is not much added with this new title bar for modals.

Screenshot 2020-03-24 at 11 56 04 AM

**Pull request closed without merging**